docs/deployment/central-backup-server.rst

```diff
@@ -81,7 +81,7 @@ The options which are added to the key will perform the following:
 Due to the ``cd`` command we use, the server automatically changes the current
 working directory. Then client doesn't need to have knowledge of the absolute
 or relative remote repository path and can directly access the repositories at
-``<user>@<host>:<repo>``.
+``ssh://<user>@<host>/<repo>``.
```

Comment:
Hmm, considering the sentence in line 82/83, this needs to stay a relative-to-cwd path, just use the ssh url syntax.

Reply:
Done